nvflare/fuel/hci/tools/admin.py

```diff
@@ -100,6 +100,8 @@ def main():
         ca_cert=ca_cert,
         client_cert=client_cert,
         client_key=client_key,
+        upload_dir=admin_config.get("upload_dir"),
+        download_dir=admin_config.get("download_dir"),
```

Comment:
If the `FileTransferModule` is required for api.py,
let's make it a default just like `HACommandModule`